modules/covid_vaccine/app/services/covid_vaccine/v0/registration_service.rb

```diff
@@ -27,9 +27,17 @@ def submit(attributes, account_id)
         # TODO: error handling
         response = CovidVaccine::V0::VetextService.new.put_vaccine_registry(attributes)
         Rails.logger.info("Vetext Response: #{response}")
-        CovidVaccine::V0::RegistrationSubmission.create({ sid: response[:sid],
-                                                          account_id: account_id,
-                                                          form_data: attributes })
+        record = CovidVaccine::V0::RegistrationSubmission.create({ sid: response[:sid],
```

Comment:
this method is now no longer returning the `record` object which the controller wants to use to serialize a response to the user.

Reply:
doh... how come no specs are failing though Mr. Vinograd. 😂

Reply:
my bad, it was my specs for create that were not verifying anything.

Reply:
I updated some specs in a since-merged PR so maybe you can upmege and re-test and see how it does